src/time.cr

```diff
@@ -409,7 +414,7 @@ struct Time
               second
 
     # Normalize internal representation to UTC
-    seconds = seconds - zone_offset_at(seconds, location)
+    seconds = seconds - zone_offset_at(seconds, location) if !location.utc?
```

Comment:
`unless`

Reply:
I thought you was against this type of discussions... it's neither more idiomatic, nor shorter.

Reply:
yes, it is more idiomatic.

Reply:
There is no style guide, so the stdlib is the reference - and this syntax is used in it, formats and compile.
Besides that for me `unless` rhymes with useless, can we stop having this type of conversations?

Reply:
`unless` keyword exists exactly to be used in such cases - which is way more natural to read _unless time is utc_ than _if not time is utc_ - that's a bad english. and yes, stdlib uses `unless`, so please use the language constructs in a way they're provided for.

Reply:
I'm against forcing people to use `unless`. I'll point it out if I think you forgot - but if you say "I don't want to use unless" then i'll just mark it as resolved and merge.

There's no need for this conversation.